<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QMainWindow" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="machine1Widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>130</width>
      <height>100</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="lineEdit_mac01_ip">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_mac01_ip">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IP ADDR</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_mac01_port">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>55</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>PORT</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_mac01_port">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="machine2Widget" native="true">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>130</width>
      <height>100</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="lineEdit_mac02_ip">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_mac02_ip">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IP ADDR</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_mac02_port">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>55</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>PORT</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_mac02_port">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="networkWidget" native="true">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>130</width>
      <height>140</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="lineEdit_net_ip_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_net_ip">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IP ADDR</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_net_port_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>55</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>PORT</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_net_port_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_net_port_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_net_port_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>95</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>PORT</string>
     </property>
    </widget>
   </widget>
   <widget class="QRadioButton" name="radioButton_mac1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Machine 1</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_mac2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Machine 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_net">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>13</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Network</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_window">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>195</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_window">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Window</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_file">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_file">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>195</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_file">
    <property name="geometry">
     <rect>
      <x>313</x>
      <y>190</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>430</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
